app/interfaces/api/entities/claim.rb

```diff
@@ -30,11 +30,11 @@ class Claim < BaseEntity
       private
 
       def messages_count
-        object.messages.count
+        object.messages.size
```

Comment:
The use of `size` instead of count is with the intent of avoiding `N+1` queries, since if the messages have been preload for the claims, `size` returns the total messages in memory, whereas `count` performs an actual `count` query on the database for each claim being rendered.